docs/nightly/en/getting-started/quick-start.md

```diff
@@ -0,0 +1,368 @@
+# Quick Start
+
+Before proceeding, please ensure you have [installed GreptimeDB](./installation/overview.md).
+
+This guide will walk you through creating a metric table and a log table, highlighting the core features of GreptimeDB.
+
+## Connect to GreptimeDB
+
+If your GreptimeDB instance is running on `127.0.0.1` with the MySQL client default port `4002` or the PostgreSQL client default port `4003`,
+you can connect to it using the following commands:
+
+```shell
+mysql -h 127.0.0.1 -P 4002
+```
+
+Or
+
+```shell
+psql -h 127.0.0.1 -p 4003 -d public
+```
+
+## Create tables
+
+Suppose you have a table named `grpc_latencies` that stores the gRPC latencies of your application.
+The table schema is as follows:
+
+```sql
+CREATE TABLE grpc_latencies (
+  ts TIMESTAMPNANOSECOND TIME INDEX,
```

Comment:
Why use the `nanosecond` precision? Looks like unnecessary.

Reply:
This is to introduce Influxdb writing data in Dashboard below, which needs TIMESTAMPNANOSECOND data type.

Reply:
changed to use TIMESTAMP here, and used a new table for influxdb writing in Dashboard below.

Reply:
Influxdb line protocol supports creating tables automatically. You never have to create the tables yourself when demonstrating the InfluxDB line protocol.